### ISSUE TITLE
[Button][Accessibility] Disabled button having both disabled attribute and aria-disabled="true"

### ISSUE THREAD

**Issue by ankitk94** [OfficeDev/office-ui-fabric-react#2145]
### Bug Report

- __Package version(s)__: 1.9.2

#### Describe the issue:

ARIA guideline says "Only use the aria-disabled attribute for elements that are not allowed to have a disabled attribute in HTML5", but for office fabric disabled button, we are passing both "disabled attribute" and "aria-disabled=true".

Link for guideline:
https://www.w3.org/TR/html-aria/

![image](https://user-images.githubusercontent.com/6390909/27898984-0b0f183c-6246-11e7-973e-f8b6bde227ae.png)

#### Actual behavior:

Both disabled="" and aria-disabled="true" are included for disabled button

#### Expected behavior:

Only disabled="" should be included as disabled is supported by button input.

**Comment by atneik** [comment]
Makes sense to me. It would be awesome if you could create a PR to fix this issue?

Button component can also return <a> anchor element, while making the change lets make sure that we still have aria-disabled="true" when it's an <a> element.

**Comment by ankitk94** [comment]
@atneik Raised the PR, can you take a look at it?
